src/Kdyby/Translation/PrefixedTranslator.php

```diff
@@ -72,68 +90,55 @@ public function unwrap()
 
 
 	/**
-	 * @param Nette\Templating\Template $template
+	 * @param $template
 	 * @return ITranslator
 	 */
-	public function unregister(Nette\Templating\Template $template)
```

Comment:
Also, this seems like a nice addition, but it's unrelated to the original PR. Can you please separate this to another PR?